justfile

```diff
@@ -393,7 +390,7 @@ lint:
 
 # Check links.
 check-links:
-    nix run .#nixpkgs.lychee -- --config tools/lychee/config.toml .
+    nix run .#nixpkgs.lychee -- --config tools/lychee/config-external.toml .
```

Comment:
Hm, I would expect this to do both (or be named with `-external`)

Reply:
I kept this as -external and added a name stating the workflow checks external links.

Reply:
Ah, alright